CS342MemberViews/Sources/CaitlinKunchur/HobbiesRow.swift

```diff
@@ -0,0 +1,89 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+
+import SwiftUI
+
+
+public struct HobbiesRow: View {
+    public var body: some View {
+        ZStack {
+            Color(hue: 0.5639, saturation: 0.0, brightness: 1).ignoresSafeArea()
+            VStack(alignment: .leading) {
+                Text("Hobbies").font(.custom(
+                    "GillSans",
+                    fixedSize: 25))
+                .padding(.leading, 15)
+                .padding(.top, 5)
+                .foregroundStyle(.blue.gradient)
+                
+                ScrollView(.horizontal, showsIndicators: false) {
+                    HStack {
+                        VStack {
+                            Image(systemName: "camera.circle")
+                                .resizable()
+                                .frame(width: 35, height: 35)
+                                .cornerRadius(5)
+                            Text("Photography")
+                                .foregroundColor(.primary)
+                                .font(.caption).padding([.bottom], 10)
+                        }
+                        .padding(.leading, 18)
```

Comment:
I would suggest you to move the different subviews into a separate private computed property to make the overall view body smaller and easier to read. E.g. you can move this view to a private property like this:
```swift
private var photographyView: some View {
    VStack {
         Image(systemName: "camera.circle")
             .resizable()
             .frame(width: 35, height: 35)
             .cornerRadius(5)
         Text("Photography")
             .foregroundColor(.primary)
             .font(.caption).padding([.bottom], 10)
    }
         .padding(.leading, 18)
}
```

One even better solution would be using a `ForEach` loop and a model to iterate over your views to generate them:

1. Generate a model type:
```swift
struct Hobby: Identifiable {
    let imageSystemName: String
    let text: String

    var id: String {
        text
    }
}
```

2. Create a collection of hobbies in your view:
```swift
    let myHobbies = [
        Hobby(imageSystemName: "camera.circle", text: "Photography"),
         Hobby(imageSystemName: "...", text: "..."),
    ]
```

3. Now use a ForEach to create your views:
```swift
        ForEach(myHobbies) { myHobby in 
             VStack {
                   Image(systemName: myHobby.imageSystemName)
                       .resizable()
                       .frame(width: 35, height: 35)
                       .cornerRadius(5)
                   Text(myHobby.text)
                       .foregroundColor(.primary)
                       .font(.caption).padding([.bottom], 10)
               }
                  .padding(.leading, 18)
        }
```